src/serac/physics/utilities/finite_element_state.hpp

```diff
@@ -255,6 +257,32 @@ class FiniteElementState {
   std::string                                       name_ = "";
 };
 
+// FIXME: Should these go somewhere else?
```

Comment:
I think it makes sense to keep these utility functions in the same file as the class they support.